client/js/lounge.js

```diff
@@ -462,18 +462,19 @@ $(function() {
 	var userStyles = $("#user-specified-css");
 	var settings = $("#settings");
 	var options = $.extend({
-		desktopNotifications: false,
 		coloredNicks: true,
+		desktopNotifications: false,
 		join: true,
 		links: true,
 		mode: true,
 		motd: false,
 		nick: true,
 		notification: true,
+		notifyAllMessages: false,
 		part: true,
-		thumbnails: true,
 		quit: true,
-		notifyAllMessages: false,
+		theme: $("#theme").attr("href").replace(/^themes\/(.*).css$/, "$1"), // Extracts default theme name, set on the server configuration
```

Comment:
Oh this is gross, but the cost of having a server-side default to override from the client...